protos/sources/vega/governance.proto

```diff
@@ -263,6 +263,8 @@ message ProposalTerms {
     NewTransfer new_transfer = 109;
     // Cancel a governance transfer.
     CancelTransfer cancel_transfer = 110;
+    // Proposal change for updating the state of a market.
```

Comment:
Is this to "change a proposal" or a "proposal to change/update the state of a market"?

Reply:
it's the latter, it's just following terminology of this proto to refer to the action as the "change", you'll see what I mean if you scroll up in that file.